### PR TITLE
misc: support Dart 2, prepare for 0.1.4+1 release

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
@isoos – would you review this?

File: pubspec.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ dev_dependencies:
   args: ">=1.0.0 <2.0.0"
   http: ">0.11.0 <0.12.0"
   path: "^1.5.1"
-  test: ">=0.12.18+1 <0.13.0"
+  test: ">=0.12.18+1 <2.0.0"
```
Comment (isoos) [inline pubspec.yaml:24]: Any reason no to set test to ^1.3.0 or something like that?

**Review by isoos — APPROVED** [review]
LGTM

File: pubspec.yaml (hunk 1 above)
Comment (kevmoo) [reply pubspec.yaml:24]: Still supporting Dart 1.x

**Pull request merged**